third-party/thrift/CMakeLists.txt

```
@@ -6,34 +6,63 @@
 
 function(thrifMain)
   set(name "thrift")
-  set(version "0.11.0")
-
   set(anchor_file "lib/libthrift.a")
 
   set(additional_library
     "lib/libthriftz.a"
   )
 
-  if(DEFINED PLATFORM_LINUX)
-    set(hash "32d0a6c2220114272fe6819a30b5ddfe000d9bed03f0d1fc3b6b68d438b5cc1e")
-  elseif(DEFINED PLATFORM_MACOS)
-    set(hash "11a4f8565728e73a54808bf55517014cc69597ef965c3b2b7ebeb52f6dd43c26")
-  elseif(DEFINED PLATFORM_WINDOWS)
-    set(hash "8b115b46d7fbae38eafbd50c4fabca1dd7e14a86ad52bda601ec374b44dc5373")
-
-    set(anchor_file "lib/parse.lib")
+  if(DEFINED PLATFORM_WINDOWS)
+    set(anchor_file "lib/thriftzmt.lib")
 
     set(additional_libraries
-      lib/thriftzmt.lib
       lib/thriftmt.lib
     )
-  else()
-    return()
   endif()
 
-  importThirdPartyBinaryLibrary("${name}" "${version}" "${hash}" "${anchor_file}" ${additional_libraries})
+  importThirdPartyBinaryLibrary("${name}" "${anchor_file}" ${additional_libraries})
 
   target_link_libraries("thirdparty_${name}" INTERFACE thirdparty_boost)
+
+  if(DEFINED PLATFORM_LINUX OR DEFINED PLATFORM_MACOS)
+    set(configure_options
+      -DCMAKE_CXX_FLAGS="-fvisibility=hidden -fvisibility-inlines-hidden $ENV{CXXFLAGS}"
```

```suggestion
      "-DCMAKE_CXX_FLAGS=-fvisibility=hidden -fvisibility-inlines-hidden $ENV{CXXFLAGS}"
```
Otherwise it fails the compilation of thrift, because the double quotes around the flags are carried to the compiler.